From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Timm=20B=C3=A4der?= <tbaeder@redhat.com>
Date: Thu, 10 Dec 2020 14:02:44 +0100
Subject: [PATCH] Add some missing <limits> includes

Otherwise, the builds fails with either gcc11 or recent clang.
---
 ogr/ogrsf_frmts/cad/libopencad/cadobjects.cpp | 1 +
 ogr/ogrsf_frmts/cad/libopencad/dwg/r2000.cpp  | 1 +
 third_party/LercLib/Lerc2.h                   | 1 +
 3 files changed, 3 insertions(+)

diff --git a/ogr/ogrsf_frmts/cad/libopencad/cadobjects.cpp b/ogr/ogrsf_frmts/cad/libopencad/cadobjects.cpp
index 0e3c895..94060a7 100644
--- a/ogr/ogrsf_frmts/cad/libopencad/cadobjects.cpp
+++ b/ogr/ogrsf_frmts/cad/libopencad/cadobjects.cpp
@@ -31,6 +31,7 @@
 
 #include "cadobjects.h"
 
+#include <limits>
 #include <math.h>
 #include <algorithm>
 
diff --git a/ogr/ogrsf_frmts/cad/libopencad/dwg/r2000.cpp b/ogr/ogrsf_frmts/cad/libopencad/dwg/r2000.cpp
index b500df8..51666e3 100644
--- a/ogr/ogrsf_frmts/cad/libopencad/dwg/r2000.cpp
+++ b/ogr/ogrsf_frmts/cad/libopencad/dwg/r2000.cpp
@@ -36,6 +36,7 @@
 #include <cassert>
 #include <cstring>
 #include <iostream>
+#include <limits>
 #include <memory>
 #include <string>
 
diff --git a/third_party/LercLib/Lerc2.h b/third_party/LercLib/Lerc2.h
index feec955..564f928 100644
--- a/third_party/LercLib/Lerc2.h
+++ b/third_party/LercLib/Lerc2.h
@@ -27,6 +27,7 @@ Contributors:  Thomas Maurer
 #include <algorithm>
 #include <cfloat>
 #include <cmath>
+#include <limits>
 #include <string>
 #include <typeinfo>
 #include "Defines.h"
